### PR TITLE
first and second task(RoboForm)

### PR DESCRIPTION
**Pull request by svirginia**

### CODE REVIEW

File: Virginia/capybara/features/step_definitions/TestRoboformsSteps.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Given(/^Navigate to robotform page$/) do
+  visit 'http://www.roboform.com/filling-test-shopping-cart' 
+end
+
+When(/^Fill fields$/) do |table|
+  @board = table.hashes
+  @board.each do |hash|
+  	#puts "HASH: #{hash}"
```
Comment (karimeSalomon) [inline Virginia/capybara/features/step_definitions/TestRoboformsSteps.rb]: Always try to delete all commented lines

File: Virginia/capybara/features/TestBrowsers.feature [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Feature: This is the feature to test browser
+
+Scenario: this is first scenario
+Given Navigate to test page
+When Select 'I have a car' checkbox
+When Select 'Female' radio button
```
Comment (karimeSalomon) [inline Virginia/capybara/features/TestBrowsers.feature]: Always concatenate sentences using And

File: Virginia/capybara/features/step_definitions/TestRoboformsSteps.rb (hunk 1 above)
Comment (svirginia) [inline Virginia/capybara/features/step_definitions/TestRoboformsSteps.rb]: sure, I will do

File: Virginia/capybara/features/TestBrowsers.feature (hunk 2 above)
Comment (svirginia) [inline Virginia/capybara/features/TestBrowsers.feature]: I will consider it in next implementations

**Pull request merged**